src/govuk/components/notification-banner/_index.scss

```diff
@@ -0,0 +1,87 @@
+@include govuk-exports("govuk/component/notification-banner") {
+  /* stylelint-disable */
+
+  .govuk-notification-banner {
+    @include govuk-font($size: 19);
+    border: $govuk-border-width solid $govuk-brand-colour;
+
+    &:focus {
+      outline: 4px solid $govuk-focus-colour;
+    }
+  }
+
+  .govuk-notification-banner--success {
+    border-color: govuk-colour("green");
+  }
+
+  .govuk-notification-banner--error {
+    border-color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__header {
+    padding: 0 govuk-spacing(3) govuk-spacing(1);
+    color: govuk-colour("white");
+    background-color: $govuk-brand-colour;
+
+
+    @include govuk-media-query($from: tablet) {
+      padding: (govuk-spacing(2) - $govuk-border-width) govuk-spacing(4) govuk-spacing(2);
+    }
+  }
+
+  .govuk-notification-banner--success .govuk-notification-banner__header {
+    background-color: govuk-colour("green");
+  }
+
+  .govuk-notification-banner--error .govuk-notification-banner__header {
+    background-color: $govuk-error-colour;
+  }
+
+  .govuk-notification-banner__title {
+    @include govuk-font($size: 19, $weight: bold);
+    margin: 0;
+    padding: 0;
+  }
+
+  .govuk-notification-banner__content {
+    @include govuk-font($size: 24, $weight: bold);
+    padding: govuk-spacing(3);
+
+    @include govuk-media-query($from: tablet) {
+      padding: govuk-spacing(4);
+    }
+  }
+
+  .govuk-notification-banner__link {
+    @include govuk-typography-weight-bold;
```

Comment:
Suggest including `govuk-link-common` here, which will take care of the focus style as well:

https://github.com/alphagov/govuk-frontend/blob/master/src/govuk/helpers/_links.scss#L5-L17

That'd also mean if we add any other 'global' link treatments in the future they'd be applied here as well.